presentation-timestamps.md

```diff
@@ -46,3 +46,83 @@ This would keep compatibility with what's out there today, allowing progressive
 
 When exposing paint timings, we look for the right trade-off between "UX-precise" and "interoperable".
 By exposing those as two timestamps, and making one of them optional, we give web developers the information that can help them optimize, without compromising on interoperability.
+
+## Security & privacy self review 
+
+See [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+### 01. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+It exposes timing information of a platform/OS operation, namely "VSync".
```

Comment:
This should cover "and for what purposes is that exposure necessary" - even in just a sentence (and then refer to the remainder of the explainer, since that's what it's about)

You can also mention that it's not the precise vsync time, but a coarsened approximation to it.